examples/README.md

```diff
@@ -0,0 +1,32 @@
+                    Copyright 2018 ABSA Group Limited
+                  
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+                You may obtain a copy of the License at
+               http://www.apache.org/licenses/LICENSE-2.0
+            
+     Unless required by applicable law or agreed to in writing, software
+       distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+                      limitations under the License.
+
+# Examples of Enceladus usage
+## Custom Rules Implementations and usage
```

Comment:
Would also be good to mention that example test cases can be found at: 
[/examples/src/test/scala/src/main/scala/za/co/absa/enceladus/examples/interpreter/rules/custom](https://github.com/AbsaOSS/enceladus/tree/feature/custom_rule_example/examples/src/test/scala/src/main/scala/za/co/absa/enceladus/examples/interpreter/rules/custom)
which has an extra "src/main/scala/" in there and really should be:
/examples/src/test/scala/za/co/absa/enceladus/examples/interpreter/rules/custom

Reply:
Both addressing, and moving also the examples themselves to correct folder